app/views/signInMobile.scala.html

```diff
@@ -0,0 +1,52 @@
+@import models.user.User
+@(signInForm: Form[com.mohiva.play.silhouette.api.util.Credentials], url: String = "/")(implicit env: com.mohiva.play.silhouette.api.Environment[User, _], request: RequestHeader, lang: Lang)
+
+@import views.html.bootstrap._
+
+@main("Project Sidewalk - Sign In", Some("/signInMobile")) {
+    <head>
```

Comment:
Again, what does this piece do? No need to answer if you answer above.

Reply:
same thing as above, it for some reason only works if it's in both places so I just left it

Reply:
Got rid of this as per other issue